web/template/watch.gohtml

```diff
@@ -211,6 +215,41 @@
                         @click="less = !less" x-text="less ? 'show more' : 'show less'">show less
                 </button>
             </div>
+            <!-- Video Sections -->
```

Comment:
The section picker should probably get it's own file and be embedded here to improve readability